spec/tapioca/cli/dsl_spec.rb

```diff
@@ -225,13 +225,13 @@ def body=(body); end
           Compiling DSL RBI files...
 
           Processing: Baz::Role
-          Wrote: sorbet/rbi/dsl/baz/role.rbi
+                create  sorbet/rbi/dsl/baz/role.rbi
           Processing: Job
-          Wrote: sorbet/rbi/dsl/job.rbi
+                create  sorbet/rbi/dsl/job.rbi
           Processing: Namespace::Comment
-          Wrote: sorbet/rbi/dsl/namespace/comment.rbi
+                create  sorbet/rbi/dsl/namespace/comment.rbi
           Processing: Post
```

Comment:
Should we also change these lines to `say_status(:processing, "Post", :yellow)` so that they line up better with the other output?